composer/datasets/imagenet_hparams.py

```diff
@@ -182,10 +182,12 @@ def initialize_object(self, batch_size: int, dataloader_hparams: DataLoaderHpara
                 transformation = transforms.Compose(train_transforms)
                 split = 'train'
             else:
-                transformation = transforms.Compose([
-                    transforms.Resize(self.resize_size),
-                    transforms.CenterCrop(self.crop_size),
-                ])
+                val_resize_size = self.resize_size
+                val_transforms: List[torch.nn.Module] = []
+                if val_resize_size > 0:
+                    val_transforms.append(transforms.Resize(val_resize_size))
```

Comment:
```suggestion
                val_transforms: List[torch.nn.Module] = []
                if self.resize_size > 0:
                    val_transforms.append(transforms.Resize(self.resize_size))
```

Reply:
Changed, and changed similar code in `if self.is_train` branch.